assets/script.js

```diff
@@ -82,4 +82,20 @@ if (document.location.pathname.indexOf('fullcpgrid') > -1) {
 // Current year of footer
 const currentYearSpan = document.getElementById('currentYear')
 const currentYear = new Date().getFullYear()
-currentYearSpan.innerText = currentYear
\ No newline at end of file
+currentYearSpan.innerText = currentYear
+
+// Search bar 
+function search_card() {
```

Comment:
1. camelCase is standard practice in JS function names. Replace it with `searchCard`
2. add the argument `event` to the function so it can receive the event object

`function searchCard(event) {`